config/unicorn/development.rb

```diff
@@ -21,7 +21,6 @@
 
 prometheus_port = PrometheusExporterPort.call
 $stdout.puts "=> Unicorn Prometheus endpoint http://localhost:#{prometheus_port}/metrics"
-$stdout.puts "=> Unicorn Prometheus endpoint http://localhost:#{prometheus_port}/yabeda-metrics"
```

Comment:
Don't you think it's useful to keep the endpoint for development until you remove it definitely?

Reply:
OK!